script.js

```diff
@@ -1,12 +1,78 @@
 //You can edit ALL of the code here
+let allEpisodes;
 function setup() {
-  const allEpisodes = getAllEpisodes();
+  allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.textContent = ""
+  const episodeCount= document.getElementById("episodeCount");
+  episodeCount.innerHTML= `Displaying ${episodeList.length}/73 episodes`;
+  //create select input
+  const selectEl=document.getElementById("episodeOptions");
+
+  episodeList.forEach(episode=>{
+    
+    //create body element
+    const divEl=document.createElement("div");
+    divEl.classList.add("container")
+    const episodeBtn=document.createElement("button");
+    episodeBtn.classList.add("episodeButton")
+    const episodeImg=document.createElement("img");
+    episodeImg.classList.add("episodeImg")
+    const episodeDescription=document.createElement("p");
+    episodeDescription.classList.add("episodeDescription")
+    //create option element
+    const optionEl=document.createElement("option");
+    optionEl.classList.add("optionElement")
+
+    // assign the text content
+    let epiNo= episode.number===10 ? episode.number : "0"+episode.number;
```

Comment:
You used the ternary operator in a good way, but it is really solving the entire problem? 
We need to add 0 a front of the numbers from the range 1 - 9, and everything more than 10 does not need it. 

![image](https://user-images.githubusercontent.com/54252969/196349234-7980b448-197a-46bc-bbd5-74cda28496b0.png)

Try this: 
`let epiNo = episode.number >= 10 ? episode.number : "0" + episode.number;`

![image](https://user-images.githubusercontent.com/54252969/196349316-cf8b9de2-da2c-4394-a4a6-5ca830e20667.png)